client/src/components/Form/Form.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export const Form = ({onSubmit, inputs, buttonText}) => {
+  return (
+  <form onSubmit={onSubmit}>
+    
+    {
+      inputs.map(input => (
+        <div key={input.name} className="input">
+        <label htmlFor={input.name}>{input.labelText}</label>
+        <input 
```

Comment:
per our visual test, clicking on a label doesn't focus the input but it's definitely a UI feature our users might appreciate

Reply:
I added an id property to the inputs and now the input is focused when the label is clicked.